cfgov/v1/atomic_elements/organisms.py

```diff
@@ -837,8 +839,20 @@ class FilterableList(BaseExpandable):
             required=False
         )),
     ])
+    # TODO: Delete this field, add migration to remove.
     topics = blocks.BooleanBlock(default=True, required=False,
                                  label='Filter Topics')
+    topic_filtering = blocks.ChoiceBlock(
+        choices=[
+            ('no_filter', "Don't filter topics"),
+            ('sort_alphabetically',
+                'Filter topics, sort topic list alphabetically'),
+            ('sort_by_frequency',
+                'Filter topics, sort topic list by number of results'),
+        ],
+        required=False,
```

Comment:
Is there a reason why this is not required? Leaving this unselected seems to cause the same behavior as selecting "Don't filter topics". If that option was removed, and we recommended leaving this blank if you don't want to filter, it'd simplify [this code](https://github.com/cfpb/cfgov-refresh/pull/5200/files#diff-dcf2a8703e945442ba8591d07085a645R897) because we'd know that if there was a selected value, you should filter.

Reply:
I'll make this required, so there aren't two options that do the same thing. I don't want to remove the "Don't filter topics option", since that would be worse UX. The code that assigns `show_topics` isn't complicated enough to require a change to UX.